### DISCUSSION TITLE
ojs to R

### DISCUSSION THREAD

**Discussion by actuariat** [quarto-dev/quarto-web#121] [Q&A]
I want to pass data collected by ojs (dataojs), like [ viewof dataojs = Inputs.range(..) ] to R like define_ojs does from R to ojs. 

This is crucial: but I didn't find a single mention or something.

**Comment by jjallaire** [comment]
Yes, you can do this w/ the Shiny bindings for Observable: https://quarto.org/docs/interactive/ojs/shiny.html
